stroyprombeton/models.py

```diff
@@ -41,13 +41,105 @@ def get_absolute_url(self):
         return reverse('category', args=(self.id,))
 
 
-class Product(catalog_models.AbstractProduct, page_models.PageMixin):
+class Option(models.Model):
+    """This doc page describes what is option: https://goo.gl/S4U9PG"""
+    tags = models.ManyToManyField(
+        'Tag',
+        related_name='options',
+        blank=True,
+        null=True,
+        verbose_name=_('tags'),
+    )
+    date_price_updated = models.DateField(auto_now_add=True, verbose_name=_('date price updated'))
+    code = models.BigIntegerField(null=True, blank=True, verbose_name=_('code'))
+    mark = models.CharField(default='', max_length=500, blank=True, verbose_name=_('mark'))
+    specification = models.TextField(default='', blank=True, verbose_name=_('specification'),)
+    length = models.IntegerField(null=True, blank=True, verbose_name=_('length'))
```

Comment:
`null=True -> default=0`
it might rid of such queries: `Options.objects.exclude(length__isnull=True, length=0)`
in favor of
`Options.objects.exclude(length=0)`.
I don't insist, so it is up to you

Reply:
@artemiy312 , y, the good strategy.
But we'll remove this fields in favor of Tags content.
We'll not write new code around them. So, let's leave them as is before deleting